pkger/clone_resource.go

```diff
@@ -832,11 +832,14 @@ func convertCellView(cell influxdb.Cell) chart {
 		ch.Kind = chartKindMosaic
 		ch.Queries = convertQueries(p.Queries)
 		ch.Colors = stringsToColors(p.ViewColors)
+		ch.HoverDimension = p.HoverDimension
```

Comment:
And here, should we handle empty / unset?

Reply:
No need.